src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -53,7 +53,7 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
-        return tagName.hashCode();
+        return tagName.toLowerCase().hashCode();
```

Comment:
So does this mean there can't be duplicate tags for a student?

Reply:
yes, see #266 

semantically `t1` and `T1` are the same thing so there is no reason for them to be treated differently.

the additional `.toLowerCase()` in the hash is because hashset uses both equals() and hashCode to check if the provided elements are the same

putting it here will preserve the provided value of the tag